applications/FluidDynamicsBiomedicalApplication/python_scripts/apply_windkessel_outlet_process.py

```diff
@@ -0,0 +1,138 @@
+        elif (pres_settings["variable_name"].GetString() != "PRESSURE"):
+            raise Exception("Outlet pressure settings variable_name is not PRESSURE.")
+        elif (ext_pres_settings["variable_name"].GetString() != "EXTERNAL_PRESSURE"):
+            raise Exception("Outlet external pressure settings variable_name is not EXTERNAL_PRESSURE.")
+        elif (pres_settings["value"].IsString()):
+            if (pres_settings["value"].GetString == ""):
+                raise Exception("Outlet pressure function sting is empty.")
+        elif (ext_pres_settings["value"].IsString()):
+            if (ext_pres_settings["value"].GetString == ""):
+                raise Exception("Outlet external pressure function sting is empty.")
+
+        self.R1      = settings["resistance1"].GetDouble()
+        self.R2      = settings["resistance2"].GetDouble()
+        self.C       = settings["compliance"].GetDouble()
+        p0_mmHg      = settings["initial_pressure"].GetDouble()
+        pv_mmHg      = settings["venous_pressure"].GetDouble()
+        self.echo    = settings["echo_level"].GetInt()
+        self.pressure_in_mmHg = settings["pressure_in_mmHg"].GetBool()
+
+        self.conv = 13.545*9.81 # Pressure conversion factor. It is used if pressure is provided in mmHg
+        if self.pressure_in_mmHg is True:
+            self.pv = pv_mmHg*self.conv
+            p0 = p0_mmHg*self.conv
+        else:
+            self.pv = pv_mmHg
+            p0      = p0_mmHg   # The pressure variable passed from the json file is given in Pa
+
+
+        self.previous_q1 = 0.0
+        self.current_p1 = p0 # in Pa
+
+        # Set the OUTLET flag in the outlet model part nodes and conditions
+        self.outlet_model_part = Model[pres_settings["model_part_name"].GetString()]
+        for node in self.outlet_model_part.Nodes:
+            node.Set(KratosMultiphysics.OUTLET, True)
+        for condition in self.outlet_model_part.Conditions:
+            condition.Set(KratosMultiphysics.OUTLET, True)
+
+    def ExecuteInitializeSolutionStep(self):
+
+        # Here the value to be provided to the outlet pressure is computed as the result of an ODE:
+        delta_t = self.outlet_model_part.ProcessInfo[KratosMultiphysics.DELTA_TIME]
+        t = self.outlet_model_part.ProcessInfo[KratosMultiphysics.TIME]
+
+        self.current_q1 = KratosFluid.FluidAuxiliaryUtilities.CalculateFlowRate(self.outlet_model_part)
+
+        self.modified_p1 = (1/self.C*(self.current_q1*( 1 + self.R1/self.R2) + self.R1*self.C*(self.current_q1 - self.previous_q1)/delta_t - (self.current_p1 - self.pv)/self.R2))*delta_t + self.current_p1
+
+        if self.echo > 0:
+            print('Current flow rate', self.current_q1)
+            if self.pressure_in_mmHg:
+                print('Outlet new pressure:', self.modified_p1/self.conv, " mmHg")
+            else:
+                print('Outlet new pressure:', self.modified_p1, " Pa")
```

Comment:
Please use the `Logger` class for this. Browse for `KratosMultiphysics.Logger.PrintInfo` usages to see many examples.